aiida/orm/nodes/node.py

```diff
@@ -149,6 +368,8 @@ class Node(Entity['BackendNode'], metaclass=AbstractNodeMeta):
     """
     # pylint: disable=too-many-public-methods
 
+    CLS_NODE_LINKS = NodeLinks
```

Comment:
Nitpick, should this not be a private attribute?
After all, we are trying to hide all extraneous node attributes from users

```suggestion
    _CLS_NODE_LINKS = NodeLinks
```

Reply:
Yeah, agree, will make it private